ca/ca_test.go

```diff
@@ -250,6 +250,25 @@ func TestFailNoSerial(t *testing.T) {
 }
 
 func TestIssueCertificate(t *testing.T) {
+	testCases := []struct {
+		name    string
+		setup   func(t *testing.T) (*CertificateAuthorityImpl, *mockSA)
+		subTest func(t *testing.T, ca *CertificateAuthorityImpl, sa *mockSA)
+	}{
+		{"IssueCertificate", issueCertificateSubTestDefaultSetup, issueCertificateSubTestIssueCertificate},
+		{"AllowNoCN", issueCertificateSubTestDefaultSetup, issueCertificateSubTestAllowNoCN},
+		{"ProfileSelection", issueCertificateSubTestDefaultSetup, issueCertificateSubTestProfileSelection},
```

Comment:
Should the "ProfileSelection" subtest be using `issueCertificateSubTestProfileSelectionSetup` as the `setup` function?